packages/api/test/e2e/api.calls.e2e.ts

```diff
@@ -36,6 +36,15 @@ describe('e2e api calls', () => {
     expect(block.header.hash.toString()).toEqual(blockHash.toString());
   });
 
+  it.skip('Query historical block', async () => {
```

Comment:
this test could form a new set of e2e tests which are added each runtime upgrade / release to check backwards compatibility is maintained.

Is it possible to hard-code a v36 block to decode here for this test?

Reply:
used onFinality node